testcontainers/src/core/wait/command_strategy.rs

```diff
@@ -0,0 +1,110 @@
+use std::time::Duration;
+
+use crate::{
+    core::{client::Client, error::WaitContainerError, wait::WaitStrategy, ExecCommand},
+    ContainerAsync, Image,
+};
+
+#[derive(Debug, Clone)]
+pub struct CommandStrategy {
+    expected_code: i64,
+    poll_interval: Duration,
+    command: ExecCommand,
+    fail_fast: bool,
+}
```

Comment:
It seems unchanged - we still have `expected_code` here which may contradict to `ExecCommand`

The idea was to change how it's organized and implemented
https://github.com/testcontainers/testcontainers-rs/pull/782#discussion_r2023388158

Reply:
Apologies for this overlook -- updated!

Reply:
@DDtKey Is there anything else you'd like fixed on this PR?

Reply:
@DDtKey Wanting to follow up here, is there anything else you'd like to see addressed on this PR that I may have missed?

Reply:
@DDtKey Hey just wanting to touch base and ensure that this PR doesn't get too stale.